src/command_modules/azure-cli-role/azure/cli/command_modules/role/custom.py

```diff
@@ -506,12 +506,57 @@ def _resolve_service_principal(client, identifier):
         raise CLIError("service principal '{}' doesn't exist".format(identifier))
 
 
+def _process_service_principal_creds(years, cert, create_cert, password, key_vault, cert_name,  # pylint: disable=too-many-arguments
+                                     start_date):
+    # pylint: disable=protected-access
```

Comment:
Instead of a disable protected access warning, should we just remove the referenced entity's __protected__ status, because the merely existence of this disable statement itself means the entity should not be protected?